common/src/main/java/com/wynntils/wc/utils/scoreboard/objectives/ObjectiveManager.java

```diff
@@ -117,7 +119,42 @@ public void onSegmentRemove(Segment segment, ScoreboardManager.SegmentType segme
 
     private List<WynnObjective> reparseObjectives(Segment segment) {
         List<WynnObjective> parsedObjectives = new ArrayList<>();
+
+        List<String> actualContent = new ArrayList<>();
+        StringBuilder scoreLine = new StringBuilder();
+
         for (String line : segment.getContent()) {
+            if (OBJECTIVE_PATTERN.matcher(line).matches()) {
+                actualContent.add(line);
+                continue;
+            }
+
+            if (OBJECTIVE_PATTERN_START.matcher(line).matches()) {
+                if (!scoreLine.isEmpty()) {
+                    WynntilsMod.error("ObjectiveManager: Multiple objective start without and ending:");
+                    WynntilsMod.error("Already got: " + scoreLine);
+                    WynntilsMod.error("Next line: " + line);
+                }
+
+                scoreLine = new StringBuilder(line);
+                continue;
+            }
+
+            if (!scoreLine.isEmpty()) {
+                scoreLine.append(line);
```

Comment:
We might need to have some check for multiple whitespace here. If I remember correctly, it looked like the second line were left-padded with space.

Reply:
It might be a bit finicky, but I think you should make sure that there is exactly one space between the non-whitespace contents of multiline objectives.

Reply:
It was not padding, they just added the newline after the space. I will make sure to remove duplicated spaces anyways.